### ISSUE TITLE
Another source of signatures

### ISSUE THREAD

**Issue by highoak** [EthTx/ethtx#24]
What do you think about getting function names from https://www.4byte.directory/signatures/ if abi is missing?

**Comment by tmierzwa** [comment]
yes, this is a great idea and definitely we will include it in new releases...

**Comment by kchojn** [comment]
> What do you think about getting function names from https://www.4byte.directory/signatures/ if abi is missing?

Small update: we are working on it, soon we will add support for signatures providers (4byte) =)
